### PR TITLE
Fixed bug with python path in cloud_sync

### PR DESCRIPTION
**Pull request by kenissur**

### CODE REVIEW

File: ugvc/scripts/cloud_sync.py [hunk 1]
```diff
@@ -10,10 +12,9 @@
 import os
 import sys
 
+sys.path.append(os.path.dirname(os.path.dirname(os.path.dirname(__file__))))
```
Comment (ilyasoifer) [inline ugvc/scripts/cloud_sync.py:15]: This is not needed since we moved to pip install pattern

**Review by ilyasoifer — CHANGES_REQUESTED** [review]
Do you want to add it to scripts section in setup.py? 
This will remove the need for line 6 since cloud_sync.py will be on the path

File: ugvc/scripts/cloud_sync.py (hunk 1 above)
Comment (kenissur) [reply ugvc/scripts/cloud_sync.py:15]: Makes sense, I couldn't figure out how to do that so I'll ask Doron for help. Thanks

Comment (kenissur) [reply ugvc/scripts/cloud_sync.py:15]: Finally done, but in another branch, I'll merge from there.

**Pull request closed without merging**